src/Components/popup.luau

```diff
@@ -179,31 +188,34 @@ function popup.addPopup(self, scope, props)
 		},
 	}
 	self.enabled:set(true)
-	newPopup.Parent = currentContainer
+	if currentContainer and currentContainer.Parent then
```

Comment:
This check for currentContainer is not needed as the beginning of the function already does this check